### PR TITLE
Add Olympics Channel

### PR DESCRIPTION
**Pull request by BellezaEmporium**
https://www.lyngsat.com/tvchannels/es/Olympic-Channel-USA.html

### CODE REVIEW

File: channels/es.m3u [hunk 1]
```diff
@@ -441,6 +441,8 @@ http://amaru.dyndns.org:8835/0.m3u8
 https://5940924978228.streamlock.net/8161/8161/master.m3u8
 #EXTINF:-1 tvg-id="OlympicsChannel.es" tvg-name="Olympics Channel" tvg-country="AU" tvg-language="English" tvg-logo="http://i.mjh.nz/.images/tv.7olympics.png" group-title="",Olympics Channel [Geo-blocked]
 https://i.mjh.nz/au/Sydney/tv.7olympics.m3u8
```
Comment (Dum4G) [inline channels/es.m3u:443]: doesn't work to me

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/es.m3u (hunk 1 above)
Comment (Dum4G) [inline channels/es.m3u:443]: Doesn't work with AU vpn to me

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/es.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/es.m3u:443]: Ah yes that link... Apparently it went forbidden.

Comment (LaneSh4d0w) [reply channels/es.m3u:443]: I can try to make it work again, but i'll need to see from where the link starts.

Comment (LaneSh4d0w) [reply channels/es.m3u:443]: Look at this one https://ocshls-olympicchannel.akamaized.net/ocshls/OCAUS_1.m3u8

Comment (Dum4G) [reply channels/es.m3u:443]: Access denied

Comment (LaneSh4d0w) [reply channels/es.m3u:443]: Noted, i'll take it off for the time being then.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**